content/docs/command-reference/check-ignore.md

```diff
@@ -81,10 +87,39 @@ $ dvc check-ignore -d file*
 .dvcignore:2:!file2	file2
 ```
 
-With the `--non-matching` option, non-matching `targets` will also be included
-in the list. All fields in each line, except for `<target path>`, will be empty.
+By default, only the last pattern matched would be shown. To see all the
+patterns matched, use `--all` (`-a`).
+
+```dvc
+$ dvc check-ignore -d -a file2
+.dvcignore:1:file*	file2
+.dvcignore:2:!file2	file2
+```
+
+With the `--non-matching` (`-n`) option, non-matching `targets` will also be
+included in the list. All fields in each line, except for `<target_path>`, will
+be empty.
 
 ```dvc
 $ dvc check-ignore -d -n other
 ::	other
 ```
+
+## Example: Check paths interactively or programmatically
+
+The `--stdin` option provides an interactive way to debug `.dvcignore` patterns:
+
+```dvc
+$ dvc check-ignore --stdin
+> file1
+file1
+> other
+> file2
+file2
```

Comment:
Another thing I noticed is that at least on Windows there's no `>` character:

![image](https://user-images.githubusercontent.com/1477535/89701638-6ec63e00-d8fe-11ea-981a-93e825e7383b.png)

Are they printed on other systems @karajan1001 ? Thanks

Reply:
I guess it's up to the shell, probably...